src/OpenTelemetry.Api/Metrics/LabelSet.cs

```diff
@@ -13,12 +13,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+using System.Collections.Generic;
+
 namespace OpenTelemetry.Metrics
 {
     /// <summary>
     /// Normalized name value pairs of metric labels.
     /// </summary>
     public class LabelSet
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LabelSet"/> class.
+        /// </summary>
+        /// <param name="labels">labels from which labelset should be constructed.</param>
+        public LabelSet(IEnumerable<KeyValuePair<string, string>> labels)
```

Comment:
I thought this implementation would be in the SDK and not in the APIs. @SergeyKanzhelev may advise about that.

Reply:
You are right. It should be in SDK. I will move it out.